config.go

```diff
@@ -1508,6 +1508,17 @@ func (c *Config) graphDatabaseDir() string {
 	)
 }
 
+// ImplementationConfig returns the configuration of what actual implementations
+// should be used when creating the chain control.
```

Comment:
"when creating the chain control" is this correct?

Reply:
No, that's not entirely correct in the latest version anymore. Fixed.